dataactbroker/README.md

```diff
@@ -382,6 +406,27 @@ Example output:
 }
 ```
 
+#### POST "/v1/update_user/"
+Update editable fields for the specified user. A call to this route should have JSON or form-urlencoded with keys "uid" and at least one from "status", "permissions", "is_active".
+
+Example input:
+
+```json
+{
+    "uid": 1,
+    "status": "approved",
+    "permissions": "agency_user",
+    "is_active": true
+}
+```
+
+Example output:
+
+```json
+{
+    "message": "User successfully updated"
+}
+```
 
```

Comment:
I've notified Kevin that the change_status route will no longer be used but instead use the update_user route.